.htaccess

```diff
@@ -28,9 +28,9 @@
 
 	# Redirect legacy URLs (assumes we're not redirecting other assets, for performance reasons)
 	# Note: Just works on top-level domains, not if the webroot is in a subfolder
-	RewriteCond %{REQUEST_URI} ^(.*)$
-	RewriteCond %{REQUEST_FILENAME} !-f
-	RewriteCond %{REQUEST_URI} !\.(css|gif|ico|jpg|js|png|swf|txt)$
+	# RewriteCond %{REQUEST_URI} ^(.*)$
```

Comment:
Well, Since Line 34 was originally commented out, Lines 31-33 do nothing except burn up resources checking the conditions. I am a big fan of removing stuff that isn't needed ;)